README.md

```diff
@@ -24,6 +24,11 @@ optimized versions of the same program.
 
 ### Phase 0
 
+* [Formal Verification of the Eth2.0 Specs in Dafny](https://github.com/ConsenSys/eth2.0-dafny) The objective of this project is to write a formal specification of the Eth2.0 specification in the verification-aware programming language [Dafny](https://github.com/dafny-lang/dafny/wiki).
+
+* [Fully Mechanised Proof of the Incremental Merkle tree algorithm](https://github.com/ConsenSys/deposit-sc-dafny). The source code in this repository is the **first complete formal proof of correctness** of the incremental Merkle tree algorithm used in the Deposit Smart Contract. 
```

Comment:
I'm not sure how I feel about the `first complete formal proof of correctness`. What about https://github.com/runtimeverification/verified-smart-contracts/blob/master/deposit/deposit-formal-verification.pdf ? Why is that not complete or formal?